userguide/tutorials/deploy_to_kubernetes.adoc

```diff
@@ -0,0 +1,363 @@
+= Deploy to Kubernetes
+
+This guide explains how to get started with Kill Bill deployment in Kubernetes.
+
+Kubernetes is an open-source system for managing containerized applications across multiple hosts.
+It provides basic mechanisms for deployment, maintenance, and scaling of applications.
+
+You can deploy Kill Bill on the Kubernetes cluster running locally or in the cloud using the resource configuration file shared below.
```

Comment:
I would move the configuration file to the end of the document in a section labeled Appendix A. You can mention it here and provide a linked cross-reference.

Reply:
The resource configuration file is important and it's the first thing a technical person would look for in the document. I added this file in a section.

Reply:
That's fine!